numbat/src/typed_ast.rs

```diff
@@ -567,6 +606,7 @@ impl PrettyPrint for Expression {
                     + m::space()
                     + with_parens(else_)
             }
+            DateTime(_, _) => m::text("datePlaceHolder"),
```

Comment:
to do

Reply:
I actually wasn't able to figure out how to get into this code, so I'm now just printing the debug representation of the `DateTime` object.  Does that sound OK?

Reply:
Fully removed in the latest commits